projects/sandbox/src/app/footer/footer.component.scss

```diff
@@ -44,6 +44,7 @@
 
```

Comment:
I saw that the changes done to the footer are not many, at least in this PR.
But I noticed that the footer itself does change between each theme, is that the desired behavior?
Also, a personal opinion, I don't think the colors of the footer go together with the whole color scheme in both themes of sandbox, and perhaps it's too big. Was this design agreed previously?

Reply:
For now the footer contains only the "sub-footer" of the portal, so it varies from the design in content, which has still to be decided (and which will better fill it out).  More redesign iterations are anticipated.

Ideally the theme switch would load the old footer from the shared library, but it's a very lightweight and temporary mechanism that conditionally includes css but which doesn't alter markup structure.  By the time we release issues of colour clash should be gone!